db/fixtures/miq_product_features.yml

```diff
@@ -3266,6 +3266,10 @@
       :description: Display Individual Container images
       :feature_type: view
       :identifier: container_image_show
+    - :name: Package List
```

Comment:
@dkorn you don't need to add a new feature to display list of packages under Container Image, if a user can see Container Images summary screen, they should be allowed to see list of packages under that Image.